conans/client/generators/cmake_find_package_multi.py

```diff
@@ -265,7 +279,7 @@ def filename(self):
     def content(self):
         ret = {}
         build_type = str(self.conanfile.settings.build_type).upper()
-        build_type_suffix = "_{}".format(build_type) if build_type else ""
+        build_type_suffix = "_{}".format(self.configuration.upper()) if self.configuration else ""
```

Comment:
I am not sure if it could cause problems if I have something like (corner case):
```
cmake.configurations = ["Debug", "debug", "DEBUG"]
```
do we already know If cmake configurations are case insensitive?

Reply:
Yes, I guess it could cause problems. Up to my knowledge the CMake generator expressions for configuration are case insensitive, so that definition shouldn't be done.